crates/polars-plan/src/dsl/string.rs

```diff
@@ -369,16 +369,16 @@ impl StringNameSpace {
             )))
     }
 
-    /// Return the number of characters in the string (not bytes).
-    pub fn n_chars(self) -> Expr {
+    /// Return the length of each string as the number of bytes.
```

Comment:
Can we add a "See also"?

And then in both functions explain that the bytes is cheaper e.g. O(1).

Counting the chars is O(n) as we need to traverse the data linearly.

Reply:
Added notes, references & time complexity!